src/components/FeatureViews/Accounts/ProfileCard.js

```diff
@@ -53,68 +98,75 @@ const ProfileCard = ({ user, route, studentInvite = false }) => {
     }, [invite, user]);
 
     const classes = useStyles();
+    const stripeColor = stringToColor(user.name);
```

Comment:
The only place you use this is on line 109. You could instead just call the function there and get rid of this variable.